man/loadOutput.Rd

```diff
@@ -0,0 +1,49 @@
+% Generated by roxygen2 (4.1.0): do not edit by hand
```

Comment:
is it best practice to check Rd files into the repo?  To me since it is auto generated, it should just be part of a build process.